test/xpu_api/language.support/support.types/null.pass.cpp

```diff
@@ -15,6 +15,8 @@
 
 #include "support/test_config.h"
 
+#include <oneapi/dpl/cstddef>
```

Comment:
@MikeDvorskiy  FYI: really we should include at least one of our public headers to avoid compile error like:
```C++
[ 91%] Building CXX object test/CMakeFiles/nullptr_t.pass.dir/xpu_api/language.support/support.types/nullptr_t.pass.cpp.o
cd oneDPL/build/test && /usr/share/miniconda/bin/icpx -DONEDPL_USE_DPCPP_BACKEND=0 -D_PSTL_TEST_SUCCESSFUL_KEYWORD=1 -I/oneDPL/test -I/oneDPL/include -isystem /usr/share/miniconda/include -O3 -DNDEBUG -std=c++17 -fno-fast-math -fopenmp-simd -MD -MT test/CMakeFiles/nullptr_t.pass.dir/xpu_api/language.support/support.types/nullptr_t.pass.cpp.o -MF CMakeFiles/nullptr_t.pass.dir/xpu_api/language.support/support.types/nullptr_t.pass.cpp.o.d -o CMakeFiles/nullptr_t.pass.dir/xpu_api/language.support/support.types/nullptr_t.pass.cpp.o -c /oneDPL/test/xpu_api/language.support/support.types/nullptr_t.pass.cpp
In file included from oneDPL/test/xpu_api/language.support/support.types/null.pass.cpp:19:
In file included from oneDPL/test/support/utils.h:24:
In file included from oneDPL/include/oneapi/dpl/execution:23:
In file included from /usr/lib/gcc/x86_64-linux-gnu/10/../../../../include/c++/10/execution:32:
In file included from /usr/lib/gcc/x86_64-linux-gnu/10/../../../../include/c++/10/pstl/glue_execution_defs.h:50:
In file included from /usr/lib/gcc/x86_64-linux-gnu/10/../../../../include/c++/10/pstl/algorithm_impl.h:22:
In file included from /usr/lib/gcc/x86_64-linux-gnu/10/../../../../include/c++/10/pstl/parallel_backend.h:16:
/usr/lib/gcc/x86_64-linux-gnu/10/../../../../include/c++/10/pstl/parallel_backend_tbb.h:70:10: error: no member named 'task' in namespace 'tbb'
    tbb::task::self().group()->cancel_group_execution();
    ~~~~~^
```

Reply:
FYI: created issue https://github.com/oneapi-src/oneDPL/issues/1246

Reply:
I believe this question is not the part of this test and this PR too.